#!/usr/bin/env python3
PACKAGE = "realsense_pointcloud"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("left_crop_param", double_t, 0, "Left crop",    0.0, 0.,   0.5)
gen.add("right_crop_param", double_t, 0, "Right crop",    0.0, 0.,   0.5)
gen.add("top_crop_param", double_t, 0, "Top crop",    0.0, 0.,   0.5)
gen.add("bottom_crop_param", double_t, 0, "Bottom crop",    0.0, 0.,   0.5)


exit(gen.generate(PACKAGE, "crop_node", "Reconfig_params"))
